core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -81,6 +81,7 @@ <h1 class="oppia-exploration-h1"><span class="protractor-test-exploration-header
 {% block footer_js %}
   {{ super() }}
   <script src="/templates/dev/head/mathjaxConfig.js"></script>
+  <script src="/third_party/static/ckeditor-4.9.2/ckeditor.js"></script>
```

Comment:
Reorganize this so that the order is:
- ckeditor
- mathjaxConfig
- MathJax

Reply:
ok

Reply:
done